docs/quickstart/cross-chain-transfers.md

```diff
@@ -0,0 +1,290 @@
+
+```js
+{
+  unsignedTx: {
+    networkID: 5,
+    blockchainID: '2JVSBoinj9C2J33VntvzYtVJNZdN2NKiwwKjcumHUWEb5DbBrm',
+    outputs: [ [Object] ],
+    inputs: [ [Object], [Object] ],
+    memo: '0x41564d207574696c697479206d6574686f64206275696c644578706f7274547820746f206578706f7274204156415820746f2074686520432d436861696e2066726f6d2074686520582d436861696e',
+    destinationChain: 'yH8D7ThNJkxmtkuv2jgBa4P1Rn3Qpr4pPr7QYNfcdoS6k6HWp',
+    exportedOutputs: [ [Object] ]
+  },
+  credentials: [
+    {
+      fxID: 'spdxUxVJQbX85MGxMHbKw1sHxMnSqJ3QBzDyDYEP3h6TLuxqQ',
+      credential: [Object]
+    },
+    {
+      fxID: 'spdxUxVJQbX85MGxMHbKw1sHxMnSqJ3QBzDyDYEP3h6TLuxqQ',
+      credential: [Object]
+    }
+  ]
+}
+```
+
+#### Import the Avax Token From X-Chain to C-Chain
+
+Select the [**`examples/evm`**](https://github.com/ava-labs/avalanchejs/tree/master/examples/evm) folder to view the AvalancheJS C-Chain examples. To import AVAX to the C-Chain from the X-Chain, select [`evm/buildImportTx-xchain.ts`](https://github.com/ava-labs/avalanchejs/blob/master/examples/evm/buildImportTx-xchain.ts)
+
+Copy the [network setting from above](#network-setting) into `evm/buildImportTx-xchain.ts`.
+
+Navigate to this part of the code and ensure that the `cHexAddress`(_Your C-Chain wallet address_) and `private key` are correct:
+
+```ts
+const cHexAddress: string = "<YOUR-CCHAIN-WALLET-ADDRESS-HERE>";
+const privKey: string = "<YOUR-PRIVATE-KEY-HERE>";
+```
+
+Run the import script:
+
+```sh
+$ ts-node examples/evm/buildImportTx-xchain.ts
+```
+
+This returns:
+
+```sh
+Success! TXID: 2uQvMcPZjmPXAyvz9cdKBphDDSmnxxx3vsUrxqpj3U92hsfQcc
+```
+
+That's it! You've transferred AVAX from the X-Chain to C-Chain!
+
+You can verify this TX by copy / pasting the import TXID into [Avascan](https://testnet.avascan.info/blockchain/c/tx/2uQvMcPZjmPXAyvz9cdKBphDDSmnxxx3vsUrxqpj3U92hsfQcc).
+
+### Transfer from the C-Chain to the X-Chain
+
+To return the AVAX back to the X-Chain, you need to do the transfer in the opposite direction.
+
+#### Export the Avax Token From C-Chain to X-Chain
+
+Select the [**`examples/evm`**](https://github.com/ava-labs/avalanchejs/tree/master/examples/evm) folder to view the AvalancheJS C-Chain examples. To export AVAX from the X-Chain to the C-Chain, select [`evm/buildExportTx-xchain-avax.ts`](https://github.com/ava-labs/avalanchejs/blob/master/examples/evm/buildExportTx-xchain-avax.ts).
+
+Make necessary changes as above for private key and network settings.
+
+You can change the amount of AVAX to send by editing the _BN_ variable: `avaxAmount`. The sample code assigns this as `1e7` or `10000000` (0.01 AVAX)
+
+The fee here will only be for exporting the asset. The import fees will be deducted from the UTXOs present on the Exported Atomic Memory, a memory location where UTXOs stay after getting exported but before being imported.
+
+```ts
+let avaxAmount: BN = new BN(1e7);
+let fee: BN = baseFee.div(new BN(1e9));
+fee = fee.add(new BN(1e6));
```

Comment:
These code does not match anymore https://github.com/ava-labs/avalanchejs/blob/master/examples/evm/buildExportTx-xchain-avax.ts#L46

Reply:
Right

We are blocked by the merge to master

the changes are in development https://github.com/ava-labs/avalanchejs/blob/c7d47ef89dc32b8415d9d41dd686f4038199921a/examples/evm/buildExportTx-xchain-avax.ts#L46

Reply:
which PR is this?

Reply:
Yes, this is the one that fixes the issue

Reply:
Does this mean that this doc PR needs to wait until after next AvalancheJS release?